applications/hub/targets/linux.x86/AvaHiMDNS.cxx

```diff
@@ -126,7 +126,8 @@ void mdns_publish(const char *name, uint16_t port)
 
     int result = avahi_entry_group_add_service(group, AVAHI_IF_UNSPEC,
         AVAHI_PROTO_UNSPEC, (AvahiPublishFlags)0, name,
-        openlcb::TcpDefs::MDNS_SERVICE_NAME_GRIDCONNECT_CAN, NULL, NULL, port,
+        openlcb::TcpDefs::MDNS_SERVICE_NAME_GRIDCONNECT_CAN,
+        openlcb::TcpDefs::MDNS_PROTOCOL_TCP, NULL, port,
```

Comment:
I'm not convinced this is equivalent. The documentation says
`/**< The domain to register this domain in. We recommend to pass NULL here, to let the daemon decide */,`
and normally the domain is ".local" I think.

Reply:
you are right, I missed that note. It was supposed to go into type and not domain. I'm thinking it may make sense to go with the option you suggested above of having two defines.